materials.js

```diff
@@ -1,4 +1,3 @@
-/* exported materials */
```

Comment:
The `exported materials` comment is important for ESLint to not mark the `materials` variable as unused. Please retain.